docs/preview/continuous-patching/README.md

```diff
@@ -0,0 +1,468 @@
+```
+
+If scan is currently running and NO patched image exists
+```sh
+image: import:dotnetapp-manual
+        scan status: Running
+        scan date: 2024-09-13 21:05:58.841962+00:00
+        scan task ID: dt21
+        patch status: ---Not Available---
+        patch date: ---Not Available---
+        patch task ID: ---Not Available---
+        last patched image: ---Not Available---
+        workflow type: continuouspatchv1
+```
+
+If patch is currently running and a patched image exists
+```sh
+image: import:dotnetapp-manual
+        scan status: Succeeded
+        scan date: 2024-09-13 21:05:58.841962+00:00
+        scan task ID: dt21
+        patch status: Running
+        patch date: 2024-09-13 21:07:32.841962+00:00
+        patch task ID: xyz2
+        last patched image: import:dotnetapp-manual-patched
+        workflow type: continuouspatchv1
+```
+
+If patch is currently running and NO patched image exists
+```sh
+image: import:dotnetapp-manual
+        scan status: Succeeded
+        scan date: 2024-09-13 21:05:58.841962+00:00
+        scan task ID: dt21
+        patch status: Running
+        patch date: 2024-09-13 21:07:32.841962+00:00
+        patch task ID: xyz2
+        last patched image: ---Not Available---
+        workflow type: continuouspatchv1
+```
+
+## Canceling Running Tasks
+
+Sometimes you may need to cancel tasks which are currently running or waiting to run. For this purpose, please run the following CLI command:
+```sh
+az acr supply-chain workflow cancel-run -r <registryname> -g <resourcegroup> --type <continuouspatchv1>
+```
+
+This command will cancel all CSSC tasks within the registry with a status of “Running”, “Queued” and “Started”. The command will output a success or failure. Failure results will follow the failure pattern of the other workflow commands if the input is incorrect.
+Running the cancel command will only affect tasks in the current schedule. For example, if a user has their schedule for 1d, and runs the cancel command, tasks in those 3 states will be canceled for that day, but will be requeued for the next day. If the schedule was a week, then that week’s tasks would be canceled, but the following week would have the tasks requeued. The common scenario for using this command is to cancel all running tasks in case a user misconfigures their continuous patching workflow and doesn’t want to wait for all tasks to finish running.
+
+## Tags
+
+- Tags suffixed with -1 to -999 are considered patch tags. Original tag and next patch is determined by splitting on the last occurance of '-'.
+- Tags suffixed with -x where x > 999 are considered original tags and their patches will be created suffixed with -x between 1 to 999. For eg: jammy-20240530 will be considered as original because even though it ends with a number, 20240530 is greater than 999. Its first patch will be created with tag jammy-20240530-1 and so on.
+- Logic is the same when tags = * or when tags are explicitly specified
```

Comment:
Customers will not be clear on what this means. I know this is the patch JSON config but there is no mention of that. Also some bullet points have periods, while some don't.